demisto_sdk/commands/content_graph/strict_objects/release_notes_config.py

```diff
@@ -0,0 +1,12 @@
+from typing import List, Optional
+
+from pydantic import Field
+
+from demisto_sdk.commands.common.constants import MarketplaceVersions
+from demisto_sdk.commands.content_graph.strict_objects.common import BaseStrictModel
+
+
+class StrictReleaseNotesConfig(BaseStrictModel):
+    breaking_changes: Optional[bool] = Field(None, alias="breakingChanges")
```

Comment:
isn't this always `True`? 🤔 (I mean, why have a json if it's not breaking?)

Reply:
Makes sense, changing it to required and all files are passed